test/sampling.jl

```diff
@@ -1,4 +1,5 @@
 using ADRIA
+using Distributions
```

Comment:
I remembered that if you don't want to specify `Distributions` as a direct dependency, you could pull it in via ADRIA, like so:

```julia
using ADRIA.Distributions
```